presets/no-esm.json

```diff
@@ -21,10 +21,22 @@
         "yn"
       ]
     },
+    {
+      "allowedVersions": "<5.0.0",
+      "matchDatasources": ["npm"],
+      "description": [
+        "Do not migrate to Material-UI v5",
+        "https://github.com/backstage/backstage/issues/7094"
+      ],
+      "matchPackageNames": [
+        "@mui/material",
+        "@mui/icons-material"
+      ]
+    },
```

Comment:
This should be IMO a new preset.  Users can already been partially migrated to v5 and because of this they wouldn't receive updates for it anymore.